test/mod/test_buildroot.py

```diff
@@ -135,5 +143,41 @@ def test_selinuxfs_ro(self):
                    "/sys/fs/selinux",
                    "ro"]
 
-            r = root.run(cmd, monitor, readonly_binds=ro_binds)
+            r = root.run(cmd, monitor, timeout, readonly_binds=ro_binds)
             self.assertEqual(r.returncode, 0)
+
+    def test_timeout(self):
```

Comment:
Please explain in the commit message how this test works and what it is verifying.